installers/git/installer.sh

```diff
@@ -6,19 +6,33 @@ APT_GET_CMD=$(which apt-get) # apt package manager for Ubuntu & other Debian bas
 PACMAN_CMD=$(which pacman) # pacman package manager for ArchLinux
 APK_CMD=$(which apk) # apk package manager for Alpine
 
+USER="$(id -un 2>/dev/null || true)"
+PREFIX=''
+if [ "$USER" != 'root' ]; then
+	if command_exists sudo; then
+		PREFIX='sudo'
+	else
+		cat >&2 <<-'EOF'
+		Error: this installer needs the ability to run commands as root.
+		We are unable to find "sudo"  available to make this happen.
```

Comment:
"We are unable to find "sudo". Make sure its available to make this happen"

Reply:
Thank you sir! I fixed it as you suggested!